kyuubi-common/src/main/scala/org/apache/kyuubi/config/ConfigBuilder.scala

```diff
@@ -136,7 +136,10 @@ private[kyuubi] case class TypedConfigBuilder[T](
   /** Checks if the user-provided value for the config matches the validator. */
   def checkValue(validator: T => Boolean, errMsg: String): TypedConfigBuilder[T] = {
     transform { v =>
-      if (!validator(v)) throw new IllegalArgumentException(errMsg)
+      if (!validator(v)) {
+        throw new IllegalArgumentException(s"The value of ${parent.key} is invalid," +
```

Comment:
We can refer to [SPARK-35456](https://issues.apache.org/jira/browse/SPARK-35456), also output value.

nit space  `invalid,` ->`invalid, `

Reply:
Thanks, updated.